README.md

```diff
@@ -1,15 +1,15 @@
 # Synthèses de l'EPL
 
-| **Documentation**  | **PDF** | **Chat** | **Forum** | **Git** |
-|:------------------:|:-------:|:--------:|:---------:|:-------:|
-| [<img src="https://cdn.pixabay.com/photo/2013/04/01/21/32/reading-99244_960_720.png" width="48">][doc-url] | [<img src="https://image.freepik.com/free-icon/pdf-file-format-symbol_318-45340.jpg" width="48">][pdf-url] | [![Join the chat at https://gitter.im/Gp2mv3/Syntheses](https://badges.gitter.im/Gp2mv3/Syntheses.svg)][chat-url] | [<img src="http://forum-epl.be/images/avatars/gallery/Photos/logo_phpBB.gif" width="96">][forum-url] | [<img src="https://gitforwindows.org/img/git_logo.png" width="48">][git-url]
+| **Documentation**  | **PDF** | **Chat** |**Git** | **Build** | **Discord**|
+|:------------------:|:-------:|:--------:|:------:|:---------:|:----------:|
+| [<img src="https://cdn.pixabay.com/photo/2013/04/01/21/32/reading-99244_960_720.png" width="48">][doc-url] | [<img src="https://image.freepik.com/free-icon/pdf-file-format-symbol_318-45340.jpg" width="48">][pdf-url] | [![Join the chat at https://gitter.im/Gp2mv3/Syntheses](https://badges.gitter.im/Gp2mv3/Syntheses.svg)][chat-url] | [<img src="https://gitforwindows.org/img/git_logo.png" width="48">][git-url] | [<img src="https://github.com/Tfloow/Syntheses/actions/workflows/main.yml/badge.svg" width="128">][build-url] | [<img src="https://upload.wikimedia.org/wikipedia/fr/thumb/4/4f/Discord_Logo_sans_texte.svg/106px-Discord_Logo_sans_texte.svg.png?20210527094128" width="64">][discord-url]
```

Comment:
On peut ajouter discord et mais garder le forum-epl. Même si c'est des années antérieures, on peut y retrouver des sujets qui sont encore utiles.

Reply:
A priori le nom de domaine forum-epl.be est libre de ce que je vois. 
![image](https://github.com/Gp2mv3/Syntheses/assets/42278077/91be9edb-85d3-4034-82e9-7430f3b7403f)
Donc je suppose que c'est inutile de garder ce lien dans le README, à moins que quelqu'un ne le reprenne ?

Reply:
Oui, je n'avais pas vu que le forum-epl était down :'( On peut le retirer du coup en effet